src/openms/thirdparty/evergreen/src/Evergreen/evergreen.hpp

```diff
@@ -0,0 +1,53 @@
+#ifndef _EVERGREEN_HPP
+#define _EVERGREEN_HPP
+
+
+// Create a macro for ALWAYS_INLINE (different modifiers on different compilers)
+#if defined(_MSC_VER)
+#define EVERGREEN_ALWAYS_INLINE __forceinline
+#else
+#define EVERGREEN_ALWAYS_INLINE inline __attribute__((always_inline))
+#endif
+
+// for convenience:
+#include "../Utility/vector_ostream.hpp"
+
+
+// Can be commented out for greater performance; these simply verify
+// shapes of things and will not be very expensive (they are not like
+// bounds checking). They should be left as-is by default, because
+// they will warn of problems that could be tricky for novice users.
+#ifdef NDEBUG
```

Comment:
@jpfeuffer how will this work with our build system?

Reply:
I think NDEBUG is set by default by CMake when in Release mode so it should work.
I can double check.